### DISCUSSION TITLE
Warning when open *.CTB

### DISCUSSION THREAD

**Discussion by wells5150** [sn4k3/UVtools#786] [Q&A] [answered]
### System

```Text
UVtools v4.0.3 X64
Operative system: Microsoft Windows 10.0.22621 X64
Graphic card: 
Processor: Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz
Processor cores: 8
Memory RAM: 5.19 / 11.80 GB
Runtime: win10-x64
Framework: .NET 6.0.23
AvaloniaUI: 11.0.5
OpenCV: 4.8.0

Path:       C:\Program Files\UVtools\
Executable: C:\Program Files\UVtools\UVtools.exe
Loaded file: UVtools_demo_file.sl1 [Version: 0] [Class: SL1File]
```

### Printer and Slicer

- Printer: Phrozen Sonic Mighty 8k
- Slicer: Lychee

### Description of the bug

UVtools gives a warning when open *.CTB files from Lychee. States the software used to create the file is not appreciated.

**Comment by sn4k3** [comment, marked as answer]
It's not related to slicer but the file format itself.
Warning will pop on every 10th file open as far I recall the limit.

**Reply by wells5150** [reply]
Thank you. for the info.
